Lib/test/test_syntax.py

```diff
@@ -596,19 +608,25 @@ def test_assign_call(self):
     def test_assign_del(self):
         self._check_error("del f()", "delete")
 
-    def test_global_err_then_warn(self):
-        # Bug #763201:  The SyntaxError raised for one global statement
-        # shouldn't be clobbered by a SyntaxWarning issued for a later one.
+    def test_global_err_first(self):
```

Comment:
I think it is better to name this test `test_global_param_err_first` since it is related to using `global` with a parameter.